maximus-two-wilfinitlike.gmail.com/prefs.js

```diff
@@ -0,0 +1,218 @@
+const Gio = imports.gi.Gio;
+const Gtk = imports.gi.Gtk;
+const GObject = imports.gi.GObject;
+
+const Lang = imports.lang;
+
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+
+const MaximusTwoSettingsWidget = new GObject.Class({
+  Name: 'MaximusTwo.Prefs.MaximusTwoSettingsWidget',
+  GTypeName: 'MaximusTwoSettingsWidget',
+  Extends: Gtk.Notebook,
+
+  IGNORABLE_APPS: "ignorable-apps", 
```

Comment:
prefs.js is a widget loaded by gnome-tweak-tool. it has nothing to do with extension.js or decoration.js